Dockerfile-gradle-with-tc

```diff
@@ -0,0 +1,2 @@
+FROM gradle:6.9.0-jdk8 AS gradle-build
```

Comment:
might be worth using 7.1.1-jdk11 (but wait till the end, I think this file will be removed).

Reply:
I was originally using latest and it complained about some version mismatch. Will see how high it can go. And yes you are correct we can remove this file now and move it directly to the compose file. Much simpler.

Reply:
if this doesn't work  - don't stress, but we wanted to move to Java11 soon, as there's huge quality of life improvements over java8

Reply:
looks like that version worked :D